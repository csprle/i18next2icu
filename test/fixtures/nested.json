{
  "user": {
    "profile": {
      "name": "{{username}}",
      "email": "{{email}}"
    },
    "settings": {
      "language": "Language: {{lang}}"
    }
  },
  "notification_zero": "No notifications",
  "notification_one": "{{count}} notification",
  "notification_other": "{{count}} notifications"
}
