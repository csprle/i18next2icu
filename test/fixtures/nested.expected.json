{
  "user": {
    "profile": {
      "name": "{username}",
      "email": "{email}"
    },
    "settings": {
      "language": "Language: {lang}"
    }
  },
  "notification": "{count, plural, =0{No notifications} one{{count} notification} other{{count} notifications}}"
}
